timber/src/main/java/timber/log/Timber.java

```diff
@@ -234,9 +234,13 @@ private Timber() {
 
   /** A {@link Tree} for debug builds. Automatically infers the tag from the calling class. */
   public static class DebugTree implements TaggedTree {
+    /** @see <a href="http://stackoverflow.com/a/8899735" /> */
+    private static final int ANDROID_LOG_ENTRY_MAX_LENGTH = 4000;
     private static final Pattern ANONYMOUS_CLASS = Pattern.compile("\\$\\d+$");
     private static final ThreadLocal<String> NEXT_TAG = new ThreadLocal<String>();
 
+    private int logEntryMaxLength;
```

Comment:
There's no value in making this not a constant. Just use really long strings in your tests.
```java
Timber.d(repeat('a', 4000) + repeat('b', 4000));
assertThat(log.get(0).msg).isEqualTo(repeat('a', 4000));
// ...
```